pkg/report/linux.go

```diff
@@ -47,11 +48,12 @@ func ctorLinux(target *targets.Target, kernelSrc, kernelObj string, ignores []*r
 		}
 	}
 	ctx := &linux{
-		kernelSrc: kernelSrc,
-		kernelObj: kernelObj,
-		vmlinux:   vmlinux,
-		symbols:   symbols,
-		ignores:   ignores,
+		kernelSrc:      kernelSrc,
+		kernelBuildSrc: kernelBuildSrc,
```

Comment:
Now that we have this, please also remove the stripPrefix function in linux.go. It tried to guess the kernelBuildSrc, but now that we have it explicitly, we don't need these hacks.